### PR TITLE
LA: Cleanup of a helper class LAExpression

### PR DESCRIPTION
**Pull request by blishko**
Includes formatting changes, some code modernisation and removal of unused methods.

### CODE REVIEW

**Comment by blishko** [comment]
I recommend to hide whitespace differences when looking at the diff.

File: src/simplifiers/LA.cc [hunk 1]
```diff
@@ -151,43 +143,40 @@ void LAExpression::print(ostream & os) const {
...
             else
-                os << " (* " << term.second << " " << logic.printTerm(term.first) << ")";
+                os << " (* " << factor << " " << logic.printTerm(var) << ")";
         }
         os << ")";
     }
     if (r == OP::EQ || r == OP::LEQ)
         os << " " << constant << ")";
-    os << '\n';
+    os << std::endl;
```
Comment (blishko) [inline src/simplifiers/LA.cc:156]: This is the only thing I would probably not do.
If you are passed a stream, it does not feel like the responsibility of this function to flush the stream.
Maybe the caller does not want that. I would keep `\n` here.

**Review by blishko — COMMENTED** [review]
Nice additional cleanup!

File: src/simplifiers/LA.cc (hunk 1 above)
Comment (aehyvari) [reply src/simplifiers/LA.cc:156]: Ok, I'll take it off!

**Review by aehyvari — APPROVED** [review]

**Comment by aehyvari** [comment]
> Nice additional cleanup!

Thanks!  Approved.

**Pull request merged**